package.json

```diff
@@ -25,17 +25,18 @@
     "happy-dom": "^7.7.0",
     "husky": "^8.0.2",
     "lint-staged": "^13.0.4",
+    "msw": "^0.49.1",
     "postcss-custom-properties": "^13.0.0",
     "prettier": "^2.8.0",
     "vitest": "^0.25.3"
   },
   "dependencies": {
     "@axodotdev/fringe": "^0.0.4",
     "@headlessui/vue": "^1.7.4",
-    "@heroicons/vue": "^2.0.13",
     "@nuxtjs/google-fonts": "^2.0.0",
     "@nuxtjs/tailwindcss": "^6.1.3",
     "@tanstack/vue-query": "^4.18.1",
+    "axios": "^1.2.1",
```

Comment:
I think it is probably better not to swap out libraries and then revert it. That's a lot of doubled work. Surely people are testing Nuxt 3 apps without using Axios and we can figure out how, too. (Or understand why it is such a problem.) If we break up this PR so that the notes functionality is disconnected from the tests, we can take a day to see if we can figure it out before giving up.